addons/website_sale_wishlist/models/product_wishlist.py

```diff
@@ -74,3 +74,12 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     wishlist_ids = fields.One2many('product.wishlist', 'partner_id', string='Wishlist', domain=[('active', '=', True)])
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.multi
+    def _is_in_wishlist(self):
```

Comment:
It's not just a move, with the current code if a variant is archived but it was in cart, it will say it's not in cart anymore, even though it's still in cart. This is a fix for it.